protocol/avdl/keybase1/metadata.avdl

```diff
@@ -41,10 +41,15 @@ protocol metadata {
          KeyBundle ReaderBundle;
   }
 
+  record LockContext {
+    long    requireLockID;
```

Comment:
I guess we could have a function in `keybase1/extras.go` that generates this lock ID, so that caller doesn't need to worry about it.